workflow/hosts/orion.yaml

```diff
@@ -22,12 +22,10 @@ ARCHCOM_TO: 'globus_hpss'
 ATARDIR: '/NCEPDEV/${HPSS_PROJECT}/1year/${USER}/${machine}/scratch/${PSLOT}'
 MAKE_NSSTBUFR: 'NO'
 MAKE_ACFTBUFR: 'NO'
-DO_TRACKER: 'NO'
-DO_GENESIS: 'NO'
 DO_AWIPS: 'NO'
 SUPPORTED_RESOLUTIONS: ['C1152', 'C768', 'C384', 'C192', 'C96', 'C48']
-COMINecmwf: /work/noaa/global/glopara/data/external_gempak/ecmwf
-COMINnam: /work/noaa/global/glopara/data/external_gempak/nam
-COMINukmet: /work/noaa/global/glopara/data/external_gempak/ukmet
-AERO_INPUTS_DIR: /work2/noaa/global/wkolczyn/noscrub/global-workflow/GEFS_ExtData/20250310
+COMINecmwf: '/work2/noaa/global/role-global/data/external_gempak/ecmwf'
+COMINnam: '/work2/noaa/global/role-global/data/external_gempak/nam'
+COMINukmet: '/work2/noaa/global/role-global/data/external_gempak/ukmet'
+AERO_INPUTS_DIR: /scratch1/NCEPDEV/global/glopara/data/GEFS_ExtData/20250310
```

Comment:
This AERO_INPUTS_DIR is incorrect, but I'm not sure what it's supposed to be.  The directory it's changed from (which is what is in PR#3455) doesn't exist.

Reply:
Same for hercules